### PR TITLE
Credits :)

### PR DESCRIPTION
**Pull request by mauricewillemsen**
Added a credits screen.
Added a button to mainmenu.

### CODE REVIEW

**Comment by peace-4-you** [comment]
So what happened to the App test ?

File: bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+/**
+ * Generates a Screen with the game credits.
+ * @author Maurice Willemsen
+ */
+public class CreditsScreen extends Screen {
+    /**
+     * All members in a string array.
+     */
+    private static final String[] MEMBERS = {"Winer Bao", "Calvin Nhieu",
+            "Justing Segond", "Maurice Willemsen", "Jason Xie"};
```
Comment (defnoob) [inline bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java]: Justin*

File: bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/*
+ * File: HighscoreScreen.java
+ *
```
Comment (defnoob) [inline bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java]: Copy pasta

Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java]: fixed

File: bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java (hunk 1 above)
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/screen/CreditsScreen.java]: fixed

**Pull request merged**